portfolio.html

```diff
@@ -0,0 +1,100 @@
+                    <ul>
+                        <li class="col3">
+                            <div class="illustration">
+                                <img src="images/monkey-peach.jpg">
+                                <h5 class="centered">Monkey</h5>
+                            </div>
+                        </li>
+                        <li class="col3">
+                            <div class="illustration">
+                                <img src="images/tower-dribble.gif">
+                                <h5 class="centered">Dragon Chase</h5>
+                            </div>
+                        </li>
+                        <li class="col3">
+                            <div class="illustration">
+                                <img src="images/mouse-cute.jpg">
+                                <h5 class="centered">Mouse</h5>
+                            </div>
+                        </li>
+                        <li class="col3">
+                            <div class="illustration">
+                                <img src="images/music-duck.jpg">
+                                <h5 class="centered">Music Duck</h5>
+                            </div>
+                        </li>
+                        <li class="col3">
+                            <div class="illustration">
+                                <img src="images/music-panda.jpg">
+                                <h5 class="centered">Music Panda</h5>
+                            </div>
+                        </li>
+                        <li class="col3">
+                            <div class="illustration">
+                                <img src="images/music-fox.jpg">
+                                <h5 class="centered">Music Fox</h5>
+                            </div>
+                        </li>
+                        <li class="col3">
+                            <div class="illustration">
+                                <img src="images/music-dino.jpg">
+                                <h5 class="centered">Music Dino</h5>
+                            </div>
+                        </li>
+                        <li class="col3">
+                            <div class="illustration">
+                                <img src="images/mouse-skip.gif">
+                                <h5 class="centered">Mouse Skip</h5>
+                            </div>
+                        </li>
+                        <li class="col3">
+                            <div class="illustration">
+                                <img src="images/music-pig.jpg">
+                                <h5 class="centered">Music Pig</h5>
+                            </div>
+                        </li>
+                    </ul>
+                </div>
+
+
+            </section>
+        </main>
+
+        <footer>
+            <section class="footer-wrapper">
+                &copy; 2021 Jen Lim      
+            </section>
```

Comment:
Since this is the only thing in the `footer` you can probably get away with not having the section.  Also generally sections should have a header tag right underneath them as the 1st child, for accessibility.